build.gradle

```diff
@@ -3,26 +3,329 @@ buildscript {
+    description = 'Update the file on S3 containing the latest version'
+    ['java', 'android'].each() { link ->
+        dependsOn "upload${link.capitalize()}LatestLink"
+    }
+    commandLine 's3cmd', 'put', "${rootDir}/version.txt", 's3://static.realm.io/update/java'
+}
+
+task distribute {
+    group = 'Release'
+    description = 'Distribute release artifacts to Bintray and S3'
+    dependsOn uploadUpdateVersion
+}
+
+def commonPom = {
+    licenses {
+        license {
+            name 'The Apache Software License, Version 2.0'
+            url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+            distribution 'repo'
+        }
+    }
+    issueManagement {
+        system 'github'
+        url 'https://github.com/realm/realm-java/issues'
+    }
+    scm {
+        url 'scm:https://github.com/realm/realm-java'
+        connection 'scm:git@github.com:realm/realm-java.git'
+        developerConnection 'scm:git@github.com:realm/realm-java.git'
+    }
+}
+
+publishing {
+    publications {
+        library(MavenPublication) {
+            artifactId = 'realm-android'
+            artifact file("${rootDir}/realm/realm-library/build/outputs/aar/realm-library-release.aar")
+            artifact file("${rootDir}/realm/realm-library/build/outputs/aar/realm-android-${currentVersion}-sources.aar")
+            artifact file("${rootDir}/realm/realm-library/build/outputs/aar/realm-android-${currentVersion}-javadoc.aar")
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'realm-android')
+                root.appendNode('description', 'Realm is a mobile database: a replacement for SQLite & ORMs.')
+                root.appendNode('url', 'http://realm.io')
+                root.appendNode('packaging', 'aar')
+                root.children().last() + commonPom
+            }
+        }
+        annotations(MavenPublication) {
+            artifactId = 'realm-annotations'
+            artifact file("${rootDir}/realm/realm-annotations/build/libs/realm-annotations-${currentVersion}.jar")
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'realm-annotations')
+                root.appendNode('description', 'Annotations for Realm: a replacement for SQLite & ORMs.')
+                root.appendNode('url', 'http://realm.io')
+                root.appendNode('packaging', 'jar')
+                root.children().last() + commonPom
+            }
+        }
+        annotationsProcessor(MavenPublication) {
+            artifactId = 'realm-annotations-processor'
+            artifact file("${rootDir}/realm/realm-annotations-processor/build/libs/realm-annotations-processor-${currentVersion}.jar")
+            pom.withXml {
+                Node root = asNode()
+                root.appendNode('name', 'realm-annotations-processor')
+                root.appendNode('description', 'Annotations processor for Realm: a replacement for SQLite & ORMs.')
```

Comment:
Same note as above: I would rewrite this to `'Annotation Processor for Realm. Realm is a mobile database: a replacement for SQLite & ORMs'`